src/slack/slack.js

```diff
@@ -100,8 +100,32 @@ export class Slack {
 
     // console.log('Upgrade complete')
 
-    // TODO: inline handleMessage
-    this.boltApp.event('message', ({event}) => this.handleMessage(event))
+    this.boltApp.event('message', async ({event, say}) => {
+      logger.info('message event')
+
+      if (event.subtype) return
+
+      if (this.amIMentioned(event)) {
+        try {
+          await this.handleUserMessage(event, say)
+        } catch (err) {
+          await say(err)
```

Comment:
this will show exact error to the user. let's stick with the "Something went wrong" with the exclamation mark emoji, just as inside `handleUserMessage`